spec/features/merchants/invoices/show_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'merchant items show page' do
```

Comment:
I still think these let/factorybot blocks are amazing

Reply:
Same @joeldenverdev !! Switching my tests over to use them now.